src/main/webapp/app/exam/participate/exercises/programming/programming-exam-submission.component.html

```diff
@@ -1,5 +1,5 @@
 <!-- ngIf online-code-editor -->
-<div *ngIf="exercise.allowOnlineEditor">
+<div *ngIf="exercise.allowOnlineEditor || examTimeline">
```

Comment:
```suggestion
<div *ngIf="exercise.allowOnlineEditor || (examTimeline && exercise.isAtLeastInstructor)">
```

Reply:
why do we need this check?

Reply:
Sorry for not elaborating directly. It's not needed, but to make the component more robust, you can think about adding this here, so any other user cannot see the online editor through some bug